data_transfer/lib/vttsma.py

```diff
@@ -1,67 +1,78 @@
-from data_transfer.config import config
-from data_transfer.services import dmpy
 from pathlib import Path
+from typing import List
 
 import boto3
 from mypy_boto3_s3.service_resource import Bucket
 
+from data_transfer.config import config
+from data_transfer.services import dmpy
+
 
 def get_bucket(creds: dict) -> Bucket:
     """
     Builds a S3 session bucket object to interface with the S3 bucket
     """
     session = boto3.session.Session(
-        aws_access_key_id=creds['aws_ak'], 
-        aws_secret_access_key=creds['aws_ask'],
+        aws_access_key_id=creds["aws_ak"],
+        aws_secret_access_key=creds["aws_ask"],
     )
-    
-    s3 = session.resource('s3')
-    bucket = s3.Bucket(creds['bucket_name'])    
-    
+
+    s3 = session.resource("s3")
+    bucket: Bucket = s3.Bucket(creds["bucket_name"])
```

Comment:
Do you think we need to move `mypy_boto3_s3` as a dev dependency since it only exists for type checking?

Reply:
My bad!